src/components/breeds/BreedItem.jsx

```diff
@@ -0,0 +1,31 @@
+import { useEffect, useState } from "react";
+import { Link, Outlet } from "react-router-dom";
+import { api } from "../../utils/api";
+
+const BreedItem = ({ breed }) => {
+  const [imageBreed, setImageBreed] = useState("");
+
+  useEffect(() => {
+    fetchBreedImage();
+  }, []);
+
+  const fetchBreedImage = async () => {
+    const url = `https://dog.ceo/api/breed/${breed}/images/random`;
+    const data = await api(url, "get");
+    const imageUrl = data.message;
+    setImageBreed(imageUrl);
+  };
+
+  const breedLink = `/${breed}`;
+
+  return (
+    <>
```

Comment:
Do you really have to use fragment here? Can you describe a use-case of react fragment?

Reply:
No it doesnt really need i forgot to drop it.  React fragment is used  when we don't want to wrap the components with extra node for example Fragment instead div